embark.el

```diff
@@ -3919,7 +3919,7 @@ and leaves the point to the left of it."
   ("$" ispell-region)
   ("f" fill-region)
   ("u" upcase-region)
-  ("d" downcase-region)
+  ("l" downcase-region)
```

Comment:
Oh my god, you're right: `d` for `downcase` was ironically *not* mnemonic at all!

Reply:
I am not sure I get your joke here, but you used "l" at another place for downcase. And we use "d" relatively consistently for delete.

Reply:
It wasn't a joke: in general Embark tries to follow Emacs's letter choices for keybindings, and Emacs uses `l` for downcasing: `M-l` and `C-x C-l` have an `l` despite being bindings for commands that start with `downcase`. So, if I followed my own guideline I would have bound this to `l` long ago.